### ISSUE TITLE
Support for ARG

### ISSUE THREAD

**Issue by hobti01** [RedCoolBeans/dockerlint#37]
### Issue details

Unfortunately the `ARG` instruction is not supported the way that `ENV` is, causing incorrect errors.

### Steps to reproduce/test case

```
FROM scratch
ARG HOME=/home/app
WORKDIR $HOME
```

```
docker run -v "$PWD/Dockerfile":/Dockerfile:ro redcoolbeans/dockerlint
...
ERROR: WORKDIR path $HOME contains undefined ENV variable on line 3
ERROR: /Dockerfile failed.
```
### Affected version(s)

0.3.1
```
        "RepoTags": [
            "redcoolbeans/dockerlint:latest"
        ],
        "RepoDigests": [
            "redcoolbeans/dockerlint@sha256:5c11597ac2ed10d2aa32db3360548b8fe9635a52078c05778d17f6359b96f2f1"
        ],
        "Parent": "",
        "Comment": "",
        "Created": "2017-08-25T06:00:29.762704377Z",
```

**Comment by hobti01** [comment]
Any chance of a new release to incorporate the fixes since April?
